services/storage/tests/unit/test_handlers_files.py

```diff
@@ -277,15 +642,54 @@ async def test_upload_real_file_with_s3_client(
     assert s3_metadata.e_tag == upload_e_tag
 
     # check getting the file actually lazily updates the table and returns the expected values
-    received_fmd: FileMetaDataGet = await get_file_meta_data(simcore_file_id)
-    assert received_fmd.entity_tag == upload_e_tag
+    # received_fmd: FileMetaDataGet = await get_file_meta_data(simcore_file_id)
+    # assert received_fmd.entity_tag == upload_e_tag
```

Comment:
ah yes, in case you do a single presigned link upload (version1), this should still work (it does), but I forgot to create a proper test for it. Good!